waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/context/CommonBeansTest.java

```diff
@@ -22,6 +22,7 @@
 
 import static com.hotels.bdp.waggledance.api.model.AbstractMetaStore.newFederatedInstance;
 
+import com.hotels.bdp.waggledance.conf.GraphiteConfiguration;
```

Comment:
Do we need this import? I don't see any code change below that uses it?

Reply:
No, we don't. Removed it.